spec/models/recipe_spec.rb

```diff
@@ -20,7 +20,9 @@
 
 RSpec.describe Recipe, type: :model do
   let(:faust) { User.create(name: "Faust") }
-  let(:recipe) { Recipe.create(name: "Eggplant Hee Hee", author: faust, difficulty: "beginner", favorite: true) }
+  let(:recipe) { Recipe.create(name: "Eggplant Hee Hee", author: faust,
+                              difficulty: "beginner", favorite: true,
+                              description: "Something tasty") }
```

Comment:
Rubocop could help with the formatting of these multi-line blocks.

Reply:
Additionally, you should look at building the habit of running rubocop on all your changed specs and `.rb` files before committing OR look into adding some kind of plugin for VS code that will automatically run rubocop for you!